### PR TITLE
add make_dummy_json.R

### PR DESCRIPTION
**Pull request by oonamcmanus**
Adding R script to generate dummy json data to feed into Highcharts ridgeline chart.

### CODE REVIEW

**Comment by hqtan** [comment]
Hi @oonamcmanus !

Thank you so much for your PR!

I installed R and ran your script locally, and was able to see 2 files generated:
- a pdf file containing the ridgeline chart
- a `..jsonmodes.json` file containing the generated json dataset

Yay!

So I the script works! :+1: 

For a moment though, I thought something went awry with the execution, when some messages about 'Conflicts' displayed when I executed the script:
```
$ Rscript make_dummy_json.R 
── Attaching packages ─────────────────────────────────────── tidyverse 1.3.1 ──
✔ ggplot2 3.3.5     ✔ purrr   0.3.4
✔ tibble  3.1.4     ✔ dplyr   1.0.7
✔ tidyr   1.1.3     ✔ stringr 1.4.0
✔ readr   2.0.1     ✔ forcats 0.5.1
── Conflicts ────────────────────────────────────────── tidyverse_conflicts() ──
✖ dplyr::filter() masks stats::filter()
✖ dplyr::lag()    masks stats::lag()

Attaching package: ‘jsonlite’

The following object is masked from ‘package:purrr’:

    flatten

Picking joint bandwidth of 1.59

```
Were those messages expected? Should I be concerned?

If that's ok, then I'm happy to approve and merge the PR!

One suggestion I would add though (possibly the subject of another PR?), would be to add some documentation on running the R script, as comments at the top of the script:
- instructions on prerequisites for running the script, eg:
  - R needs to be installed (see [this link](https://cran.rstudio.com/) for installation instructions
  - the following R packages need to be installed via the R command line:
    - `tidyverse`. **N.B** 
      - this package takes a while to install!!
      - may need a bunch of other dependencies that needs to be installed first; read the error messages carefully!
    - `ggridges`
    - ...
  - how to run script, and what to expect as a result, eg:
    - run script with this command: `Rscript make_dummy_json.R`
    - expect 2 files to be generated...
    - if you see the following messages about conflicts appear during execution, its ok ...

**Comment by hqtan** [comment]
[DEV: highcharts spike: histograms charts for distribution of commute distances by transport mode](https://trello.com/c/b99uFgaX/194-dev-highcharts-spike-histograms-charts-for-distribution-of-commute-distances-by-transport-mode)

**Comment by oonamcmanus** [comment]
Hi @hqtan

Great feedback! So those warning messages are no problem - they are just
warning that some functions in the packages you've loaded have the same
name as functions in base R; when that happens, R chooses which version to
use by default and warns you that the other one is "masked" (you can
override this by explicitly stating which package you're calling the
function from, E.g., "dplyr:: filter" instead of just "filter". TL;dr don't
worry these are common messages and don't affect anything.

HOWEVER this (and your comment that tidyverse takes a long time to install)
made me realise that I've been a bit lazy here - "tidyverse" is a big
"package of packages". Since I have the whole thing installed, and I use a
lot of the packages frequently, I just load the whole "parent package" each
time. But for this purpose it would be better if I just choose the one or
two I actually need; that way others who use the script would only need to
install those smaller packages. I'll make that change and add the
instructions for running the script.

Thanks heaps for the feedback.
Oona

**Comment by hqtan** [comment]
Thx Oona for looking into calling only the specific tidyverse packages to use in your script. A really good idea :+1: 

I'll leave this PR open till you've made that update :+1: 
No rush :slightly_smiling_face:

File: scripts/make_dummy_json.R [hunk 1]
```diff
@@ -0,0 +1,47 @@
+#This script creates "dummy data" for the Civic Makers carbon calculator project
+
+#HOW TO RUN THIS SCRIPT: 
+# 1) Install R from https://cran.ms.unimelb.edu.au/
+# 2) Uncomment the "install.packages" rows below if you don't already have these packages installed
+# 3) From Command line (set to the directory where this script is saved), run the command ` Rscript make_dummy_json.R ` 
+# 4) This should produce two outputs: a .png (?) that shows
```
Comment (hqtan) [inline scripts/make_dummy_json.R:7]: step 4 seems incomplete?

File: scripts/make_dummy_json.R [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+# 1) Install R from https://cran.ms.unimelb.edu.au/
+# 2) Uncomment the "install.packages" rows below if you don't already have these packages installed
+# 3) From Command line (set to the directory where this script is saved), run the command ` Rscript make_dummy_json.R ` 
+# 4) This should produce two outputs: a .png (?) that shows 
+
+# install.packages("ggplot2")
+# install.packages("jsonlite")
+# install.packages("ggridges")
+
+library(tidyverse)  #packages for manipulating data
```
Comment (hqtan) [inline scripts/make_dummy_json.R:13]: should we remove this line, if we no longer need `tidyverse` package?
```suggestion
```

**Review by hqtan — CHANGES_REQUESTED** [review]

**Comment by hqtan** [comment]
Nice work @oonamcmanus :raised_hands: 

removing `tidyverse` from the script makes sense, if we don't have to use it :+1: 

I noticed a couple other things in your recent edits that may need looking into? :pray:

**Review by hqtan — APPROVED** [review]

**Comment by hqtan** [comment]
Hi @oonamcmanus !

thanks for the updates! The instructions for running the script are much clearer now; I was able to follow them (using Rscript approach) to get the script to generate the 2 output files :+1: 

:bow:

**Pull request merged**